spec/system/create_generic_spec.rb

```diff
@@ -38,7 +38,6 @@
       fill_in('Creator', with: 'Doe, Jane')
       fill_in('Keyword', with: 'testing')
 
```

Comment:
why was the select removed?

Reply:
Because it was appearing under the fold. We havent made decisions what we want placed where on the form, so this test is less valid. Mostly just testing you can create a generic object with some data and it creates. So I think whats filled out is less important